models/oscr-extract.js

```diff
@@ -0,0 +1,72 @@
+
+
+// Using th schema from the CSV download available at http://www.oscr.org.uk/charities/search-scottish-charity-register/charity-register-download
+var schemas = { "v0.1": {} };
+
+schemas["v0.1"].oscr_register = {
+  // registered number of a charity
+  "Charity Number": String,
```

Comment:
It's true that this model should reflect the regulator's schema, so looks good overall, however having spaces in field names is bad form.  I'll accept for now but will soon replace with underscores.